src/containers/App.js

```diff
@@ -137,10 +137,11 @@ class App extends React.Component {
         <NavDrawer
             active={navDrawerIsVisible}
             scrollY
-            permanentAt="md"
+            permanentAt={navDrawerIsVisible ? 'lg' : null}
```

Comment:
This change broke the behavior on the entry page. What's the idea behind this one?